fillager/pom.xml

```diff
@@ -13,21 +13,16 @@
 	<description>Temporary file storage during archival of benefit applications</description>
 
 	<properties>
-		<otj-pg-embedded.version>0.13.4</otj-pg-embedded.version>
-		<embedded-postgres.version>1.3.1</embedded-postgres.version>
-		<flyway.version>8.5.7</flyway.version>
-		<vault-java-driver.version>5.1.0</vault-java-driver.version>
-
-		<konfig.version>1.6.10.0</konfig.version>
-		<coroutines.version>1.6.1</coroutines.version>
-		<springdoc.version>1.6.7</springdoc.version>
+		<otj-pg-embedded.version>1.0.1</otj-pg-embedded.version>
```

Comment:
Denne versjonen forutsetter at docker kjører lokalt for å kunne bygge og kjøre testene. Dette må dokumenteres i README. Dette gir noen utfordringer for oss som har Windows.